components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -153,20 +153,26 @@ export default class InviteMembersStep extends React.PureComponent<Props, State>
     }
 
     copyLink = () => {
-        const textField = document.createElement('textarea');
-        textField.innerText = this.getInviteURL();
-        textField.style.position = 'fixed';
-        textField.style.opacity = '0';
-
-        document.body.appendChild(textField);
-        textField.select();
-
-        try {
-            this.setState({copiedLink: document.execCommand('copy')});
-        } catch (err) {
-            this.setState({copiedLink: false});
+        const clipboard = navigator.clipboard;
```

Comment:
FYI, there's `Utils.copyToClipboard` that does this as a helper, but it currently doesn't do the `navigator.clipboard` bit we talked about yesterday, so you'd have to wait for my PR to use that. You could use the current version for the textarea case at least if you wanted though.

Reply:
Well, depending when your PR gets in I can update that in your PR/as a seperate one later. Definitely something I can do though.